features/url-formattering.feature

```diff
@@ -0,0 +1,98 @@
+#language: nl
+
+Functionaliteit: url formattering
+
+    Als consumer van de Haal Centraal APIs
+    Wil ik kunnen aangeven hoe de urls in een response moeten worden geformatteerd
+    Zodat ik de urls op mijn gewenste manier kan verwerken.
+
+    Als provider van een Haal Centraal API
+    Wil ik [URI Template](https://datatracker.ietf.org/doc/html/rfc6570) gebruiken om URIs te beschrijven
+    Zodat een verzameling van gelijksoortige urls met één URI template kan worden gerepresenteerd.
+
+    Zie de [uri formattering](../docs/uri-formattering.md) voor uitgebreide uri formattering documentatie. 
+
+    Hieronder zijn voorbeelden uitgewerkt voor de BRP Bevragen API
+ 
+Achtergrond:
+    * de Haal Centraal APIs met hun base url en base url template placeholder
+    | Naam Haal Centraal API | base url                                                         | base url template placeholder |
+    | BRP Bevragen           | https://www.haalcentraal.nl/haalcentraal/api/brp                 | {brpserverurl}                |
+    | BAG Bevragen           | https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1 | {bagserverurl}                |
+    * de persoon met burgerservicenummer "999993847"
+    * de kinderen van persoon met burgerservicenummer "999993847"
+        | burgerservicenummer kind |
+        | 999994918                |
+        | 999995121                |
+    * de verblijfplaats van persoon met burgerservicenummer "999993847"
+        | adresseerbaarobjectidentificatie |
+        | 0518010000781379                 |
+
+Rule: Met 'uri-reference' als uri format, worden 'same API links' als relatieve urls en 'externe API links' als absolute urls geformatteerd
+
+    Scenario: uri-reference is opgegeven als Accept-Uri-Format header waarde
+        Als GET "/ingeschrevenpersonen?burgerservicenummer=999993847" wordt aangeroepen met HTTP header "Accept-Uri-format"="uri-reference"
+        Dan bevat _links in de response minimaal de volgende links
+        | naam  | href                                                                                                                        | templated |
+        | base  | https://www.haalcentraal.nl/haalcentraal/api/brp                                                                            |           |
+        | self  | /ingeschrevenpersonen?burgerservicenummer=999993847                                                                         |           |
+        | adres | https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/adressen?adresseerbaarObjectIdentificatie=0518010000781379 |           |
```

Comment:
een adres in BAG halen we op met /adressen/{nummeraanduidingIdentificatie}
ook in regel 28/29 moet de nummeraanduidingIdentificatie i.p.v. de adresseerbaarobjectidentificatie worden opgegeven